runtime/src/main/scala/akka/grpc/internal/HardcodedServiceDiscovery.scala

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.internal
+
+import akka.discovery.SimpleServiceDiscovery
+import akka.discovery.SimpleServiceDiscovery.Resolved
+
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+
+class HardcodedServiceDiscovery(resolved: Resolved) extends SimpleServiceDiscovery {
```

Comment:
Is this a utility that should be provided by akka-management in the future? Can we add a ticket to track that?

Reply:
internal api markers in `internal` package